### DISCUSSION TITLE
Regarding my participation in LXQt

### DISCUSSION THREAD

**Discussion by yan12125** [lxqt/lxqt#2196] [General]
As you may already know, my participation in LXQt has been reduced a lot. I have little spare time, and I think it's time to hand out my maintainership on qtermwidget/qterminal. How about changing how pull requests are handled in qtermwidget/qterminal? For example, merge a PR as soon as one reviewer has checked the code or run it for a few weeks.

**Comment by tsujan** [comment]
LXQt devs may not be present for long periods of time. I think that's completely understandable. They're all volunteers, who give a part of their free times to LXQt. No one expects more.

> How about changing how pull requests are handled in qtermwidget/qterminal? For example, merge a PR as soon as one reviewer has checked the code or run it for a few weeks.

No one is familiar with qtermwidget as well as you are. If I find the time, I might be able to make some PRs — as I've done so far. As for reviewing others' PRs, I might be able to do so for qterminal if I find the time but I'm not sure about qtermwidget.

That being said, your idea seems good to me. If a code is reviewed or made by an LXQt member and, at least, one LXQt member have tested it, it could be merged. Otherwise — and especially when we aren't sure about side effects — we'll wait for you. Is that acceptable?

**Reply by yan12125** [reply]
> Otherwise — and especially when we aren't sure about side effects — we'll wait for you. Is that acceptable?

Sounds good!

> No one is familiar with qtermwidget as well as you are. If I find the time, I might be able to make some PRs — as I've done so far. As for reviewing others' PRs, I might be able to do so for qterminal if I find the time but I'm not sure about qtermwidget.

Yep I have once studied them. Feel free to assign me for changes related to escape sequences in qtermwidget :)

**Reply by tsujan** [reply]
OK. With qterminal itself, I might be able to help more, especially when the issues are about GUI, not the core code of qtermwidget.

**Reply by yan12125** [reply]
Thanks!
